README.md

```diff
@@ -16,7 +16,7 @@ The columns of the samplesheet are defined as follows:
 
 - sample: The sample ID name. This name should not contain spaces.
 - mlst_alleles: A URI path to a JSON-formatted genomic profile. An example of this file is provided in [tests/data/profiles/S1.mlst.json](tests/data/profiles/S1.mlst.json).
-- metadata_partition: The specific metadata column used to partition the genomic profiles. For example, this column might refer to the outbreak number and the contain such entries as "1", "2", etc.
+- metadata_partition: The specific metadata column used to partition the genomic profiles. For example, this column might refer to the outbreak number and the contain such entries as "1", "2", etc. If this entry is missing from a sample, the sample will not be included in the analysis and will be reported in the `arborator/metadata.excluded.tsv` output file.
```

Comment:
Thanks for the suggestion. I've updated it in a548fa0, so hopefully that's more clear.

I want to keep the description of the excluded in multiple places (even though it's unnecessary), because in my experience readers of documentation are sometimes a bit impatient and want to jump directly to the problematic parameters (ctrl-F, etc.) to avoid reading the whole README.